api/metadata.py

```diff
@@ -2075,12 +2081,10 @@ def metadata_attr_list(self, request):
 
             cursor.close()
             db.close()
-
+            request_finished.send(self)
             return MetadataAttrList(items=data, count=len(data))
 
         except (IndexError, TypeError):
-            if cursor: cursor.close()
-            if db: db.close()
             raise endpoints.InternalServerErrorException('Error retrieving attribute list')
         finally:
             if cursor: cursor.close()
```

Comment:
Oh, maybe it's not showing--here in the diffs all I can see is the db.close being removed from except and not added to the finally, but maybe that's just github being github-y (or it was already there).